js/app/lite.html

```diff
@@ -24,8 +24,9 @@
 		<textarea id="code-input" cols="30" rows="10">
```

Comment:
This is just a sample HTML file for local development, and I updated it to use the newly introduced APIs to mount the files and install the requirements.